src/txkube/testing/_eliot.py

```diff
@@ -5,7 +5,7 @@
 
 from fixtures import Fixture
 
-from eliot import add_destination
+from eliot import add_destination, remove_destination
```

Comment:
See https://github.com/LeastAuthority/txkube/pull/8/files#diff-58b820f05af5c1f18e36eef9bede77cbR64 for the use of this.